portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects | Peter Geertsema's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="nav-bar">
+      <ul id="nav-list">
+        <li class="nav-item"><a class="nav-inactive" href="index.html">Home</a></li>
+        <li class="nav-item"><a class="nav-inactive" href="mickey.html">Mickey</a></li>
+        <li class="nav-item"><a class="nav-active" href="projects.html">Projects</a></li>
+      </ul>
+    </div>
+    <div id="content">
+      <h1>A random selection of projects</h1>
+      <ul>
+        <li>In Computer Systems at UMD I implemented logic for a shell supporting input/output redirection,
+            piping, and subshells all in C</li>
+        <li>Last summer I scripted a system converting old PDF documentation to markdown for use in MKDocs</li>
+        <li>Last summer I also created a comprehensive two-way sync with ServiceNow in C#</li>
+        <li>In AP Computer Science I made a Java version of the asteroids game</li>
+      </ul>
+      <p>For the many smaller projects I've worked on at past internships, please take a look at my 
+        <a target="_blank" href="https://drive.google.com/file/d/1CnNiKtx5em6K1d9h9V4P7hj8xx8pJsrc/view">resume</a>.</p>
```

Comment:
for purposes of consistency, could you please move this </p> to the next new line

Reply:
Committed. Is this what you meant?
```html
<li>In AP Computer Science I made a Java version of the asteroids game</li>
      </ul>

      <p>For the many smaller projects I've worked on at past internships, please take a look at my 
        <a target="_blank" href="https://drive.google.com/file/d/1CnNiKtx5em6K1d9h9V4P7hj8xx8pJsrc/view">resume</a>.</p>
```